plugins/email_service/app/views/layouts/email_service/application.html.erb

```diff
@@ -5,7 +5,7 @@
   <%= csrf_meta_tags %>
```

Comment:
Why do you need an own application template?

Reply:
it is still needed to do pre checks while landing on the plugin.